server/src/test/java/org/allaymc/server/item/type/AllayItemTypeTest.java

```diff
@@ -24,7 +25,7 @@ void testCreation() {
         assertEquals(0, itemStack.getMeta());
         assertEquals(0, itemStack.getDurability());
         assertEquals("", itemStack.getCustomName());
-        assertEquals(NbtMap.EMPTY, itemStack.getCustomNBTContent());
+        assertTrue(itemStack.getPersistentDataContainer().isEmpty());
```

Comment:
Remove tests for old `CustomNBT` directly, as this is intended to be replaced by PDC